### ISSUE TITLE
Shear behaviour for whitelisting

### ISSUE THREAD

**Issue by AtomicBlom** [AtomicBlom/ShearMadness#11]
Add support for whitelisting which blocks are allowed to regrow.
Create an IMC API for allowing mods to whitelist their decorative blocks.

**Comment by ghost** [comment]
Yes! Cobblegen sheep! Glassgen sheep! **CAKEGEN SHEEP!** *cackles insanely* Umm...
Alternatively or additionally, a slow breeding/production cycle like the Chickens mod. Whitelist diamond sheep, but set them to take an hour to regrow a shard.
